wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController.swift

```diff
@@ -467,6 +478,58 @@ final class ConversationContentViewController: UIViewController {
         tableView.reloadRows(at: visibleRows, with: .none)
         tableView.endUpdates()
     }
+
+    // MARK: - Update Timer
+
+    @objc
+    private func startRefreshTimerIfNeeded() {
+        stopRefreshTimer()
+
+        var timeInterval = TimeInterval()
+        for indexPath in tableView.indexPathsForVisibleRows ?? [] {
+            let section = dataSource.currentSections[indexPath.section]
+            for cellDescription in section.elements {
+                if let refreshInterval = cellDescription.conversationCellModel?.refreshInterval, refreshInterval > 0 {
+                    timeInterval = timeInterval == .zero
+                        ? refreshInterval
+                        : min(timeInterval, refreshInterval)
+                }
+            }
+        }
+
+        guard timeInterval > 0 else { return }
+        print("starting refresh timer with interval: \(timeInterval)")
```

Comment:
question: are these prints still needed? If so, it would be better to log them, otherwise discard them.

Reply:
Thanks, almost missed them.
I converted it to using WireLogger.